events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -234,6 +234,20 @@ stasis_trap_outcome.5001 = {
 				STUNNER = scope:owner
 			}
 		}
+
+		if = {
+			limit = {
+				exists = scope:scheme_discovered
+			}
+			scope:owner = {
+				save_scope_as = left_portrait
+			}
+		}
+		else = {
+			scope:target = {
+				save_scope_as = left_portrait
+			}
+		}
```

Comment:
But the animation is still `scheme`. So showing the target "scheming" isn't correct.